domains/tommy-shelby.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Tommy Shelby portfolio",
+  "repo": "https://github.com/prbhanu/tommy-shelby",
+  "owner": {
+    "username": "prbhanu",
+    "email": "prreddy494@gmail.com",
+    "twitter": "@prreddy05413731"
+  },
+  "record": {
+    "CNAME": "https://prbhanu.github.io/tommy-shelby/"
```

Comment:
the cname cannot have a `/`. if you need to have a `/`, use a `URL` record instead.

Reply:
Can u check now?